requirements/common.txt

```diff
@@ -28,13 +26,10 @@ geoip2>=2.4.2,<3
 Flask-HTMLmin>=1.2,<2
```

Comment:
Not needed I guess. If at all if this is used anyway, remove those parts and don't use this anymore.